apps/web/package.json

```diff
@@ -19,6 +19,8 @@
     "eslint": "7.32.0",
     "eslint-config-custom": "*",
     "tsconfig": "*",
+    "tailwind-config": "*",
+    "tailwindcss": "^3.2.4",
```

Comment:
Is not postcss required with tailwind ? (as you create a postcss config in this repo)

Reply:
Good catch, that's why I had the hotreload issue